client/landing/stepper/declarative-flow/internals/components/survery-manager/index.tsx

```diff
@@ -18,6 +18,9 @@ const MIGRATION_SURVEY_FLOWS = [
 ];
 
 const SurveyManager = () => {
+	// Temporarily disabled, context: https://a8c.slack.com/archives/C085HCWCEDN/p1735579884005299?thread_ts=1735306693.890579&cid=C085HCWCEDN
+	return null;
```

Comment:
```suggestion
const SurveyManager = ({disabled}) => {

	if ( ! params.flow  || disabled) {
		return null;
	}

```
It is better than maintaining weird code and having to explain it.
```

Reply:
Sounds good, [updated it here](https://github.com/Automattic/wp-calypso/pull/97852/commits/917cd6d2ee2edf119ed637b58d27e1edfb0d852b).